orchagent/aclorch.cpp

```diff
@@ -1898,18 +2024,17 @@ sai_status_t AclOrch::bindAclTable(sai_object_id_t table_oid, AclTable &aclTable
     sai_status_t status = SAI_STATUS_SUCCESS;
 
     SWSS_LOG_INFO("%s table %s to ports", bind ? "Bind" : "Unbind", aclTable.id.c_str());
+    
+    if (aclTable.portSet.empty())
```

Comment:
I don't think this condition will be ever true in this flow. Can you check?

Reply:
you are correct, I removed this flow.